clients/openai-python/tests/test_openai_compatibility.py

```diff
@@ -449,44 +449,38 @@ async def test_async_json_streaming(async_client):
 
 
 @pytest.mark.asyncio
-async def test_reject_developer_and_system(async_client):
+async def test_allow_developer_and_system(async_client):
     messages = [
         {
             "role": "developer",
-            "content": [
-                {
-                    "type": "text",
-                    "tensorzero::arguments": {"assistant_name": "Alfred Pennyworth"},
-                }
-            ],
+            "content": [{"type": "text", "text": "Developer message."}],
         },
         {
             "role": "system",
             "content": [
                 {
                     "type": "text",
-                    "tensorzero::arguments": {"assistant_name": "Alfred Pennyworth"},
+                    "text": "System message.",
                 }
             ],
         },
         {
             "role": "user",
-            "content": [
-                {"type": "text", "tensorzero::arguments": {"country": "Japan"}}
-            ],
+            "content": [{"type": "text", "text": "User message."}],
         },
     ]
     episode_id = str(uuid7())
 
-    with pytest.raises(BadRequestError) as exc_info:
-        await async_client.chat.completions.create(
-            extra_headers={"episode_id": episode_id},
-            messages=messages,
-            model="tensorzero::function_name::json_success",
-        )
+    result = await async_client.chat.completions.create(
+        extra_headers={"episode_id": episode_id},
+        messages=messages,
+        model="tensorzero::model_name::dummy::echo_request_messages",
+    )
+    assert result.model == "dummy::echo_request_messages"
+    assert result.episode_id == episode_id
     assert (
-        "Invalid request to OpenAI-compatible endpoint: At most one system message is allowed"
-        in str(exc_info.value)
+        result.choices[0].message.content
+        == '{"system":"Developer message.System message.","messages":[{"role":"user","content":[{"type":"text","text":"User message."}]}]}'
```

Comment:
@GabrielBianconi do we want to put a newline in the middle or a space or some delimiter?

Reply:
`\n`